trailpanion/src/constants.js

```diff
@@ -0,0 +1 @@
+export const OPEN_WEATHER_KEY = "93a41f94d59e95525716fb821ed543fc";
```

Comment:
Not redux related but I like to keep things like this in a separate file. Though actually API keys shouldn't be kept in a public repo